### PR TITLE
unit test PR

### PR DESCRIPTION
**Pull request by Michael3161010**
unit test PR

### CODE REVIEW

**Review by Michael3161010 — COMMENTED** [review]
@Davisjames0901 
This is actually the first PR I've submitted on Github, just trying to get a feel for the platform.

**Comment by abelsayed** [comment]
@Michael3161010 can you tag me now?

**Comment by Michael3161010** [comment]
@CodingMate  yep, I can now

File: src/DependencyReviewer/Program.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                    while (!ctx.IsFinished)
+                    {
+                        task1.Increment(1.5);
+                        task2.Increment(0.5);
+                    }
+                });
+
+            AnsiConsole.MarkupLine("[blue]Now Viewing Dependency Report[/]");
+            Thread.Sleep(2000);
+            AnsiConsole.MarkupLine($"[red]There are {analysis.InScopeRegisteredTypes.Count} In Scope Registered Types[/]");
```
Comment (Michael3161010) [inline src/DependencyReviewer/Program.cs:53]: This is very quick and dirty, but I thought it'd be cool to use that Spectre.Console to output some of the results.

**Pull request merged**